libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -548,11 +566,11 @@ void AP_MotorsHeli_Single::servo_test()
 // parameter_check - check if helicopter specific parameters are sensible
 bool AP_MotorsHeli_Single::parameter_check(bool display_msg) const
 {
-    // returns false if Phase Angle is outside of range 
-    if ((_phase_angle > 90) || (_phase_angle < -90)){
+    // returns false if Phase Angle is outside of range
+    if ((_phase_angle > 30) || (_phase_angle < -30)){
         if (display_msg) {
             gcs().send_text(MAV_SEVERITY_CRITICAL, "PreArm: H_PHANG out of range");
-        }
+      }
```

Comment:
looks like a space may be needed here to align the brace with the "i" in the if statement it closes.